client/wfnews-war/src/main/angular/src/app/components/admin-incident-form/admin-incident-form.component.ts

```diff
@@ -348,6 +348,8 @@ export class AdminIncidentForm implements OnInit, OnChanges {
     };
 
     try {
+      console.log('jsonn')
```

Comment:
You can remove console logs once you're done with them

Reply:
Done, thanks